app/src/main/java/com/cdio/solitaire/fragment/CameraFragment.kt

```diff
@@ -126,8 +120,6 @@ class CameraFragment : Fragment(), SensorEventListener {
 
         // Set up the camera and its use cases
         setUpCamera()
```

Comment:
```suggestion

        // Wait for the views to be properly laid out
        fragmentCameraBinding.viewFinder.post {

            // Set up the camera and its use cases
            setUpCamera()
        }
```

This was how it was before the landscape change, I had to remove it to fix the crash.
I guess it is better to do it this way...